backend/handler/services/SandboxService.ts

```diff
@@ -0,0 +1,102 @@
+import * as request from './RequestService';
+import manager from './ManagerService';
+import { ALLOWED_METHODS_OBJECT } from '../../consts';
+
+const catchError = (err: { message: any }, serviceName: 'request' | 'manager', method: string) =>
+    `Error while reuesting ${serviceName} - ${method} - ${err.message}`;
+
+const call = async (
+    serviceName: 'request' | 'manager',
+    method: keyof typeof ALLOWED_METHODS_OBJECT,
+    url: string,
+    params = '{}'
+) => {
+    try {
+        const paramsObj = JSON.parse(params);
+        const data =
+            serviceName === 'manager'
+                ? await manager.call(
+                      ALLOWED_METHODS_OBJECT[method as keyof typeof ALLOWED_METHODS_OBJECT],
+                      url,
+                      paramsObj
+                  )
+                : await request.call(
+                      ALLOWED_METHODS_OBJECT[method as keyof typeof ALLOWED_METHODS_OBJECT],
```

Comment:
I believe there's no need to cast anymore as `method` is of `keyof typeof ALLOWED_METHODS_OBJECT` type.
```suggestion
                      ALLOWED_METHODS_OBJECT[method],
                      url,
                      paramsObj
                  )
                : await request.call(
                      ALLOWED_METHODS_OBJECT[method],
```

Reply:
Yes, right. Here too no more casting required, updated, Thanks